CHANGES

```diff
@@ -0,0 +1,17 @@
+===============================================================================
+                               Changes in 0.2
+===============================================================================
+
+# Add support for reduction operations (e.g. sum, prod, min, max, ...)
+
+# Add support for AMD GPUs via HIP backend
+
+# Add support for NVIDIA HPC SDK compilers
+
+# Add blocking pack/unpack APIs
+
```

Comment:
We also added `yaksa_pack/unpack_stream` -- https://github.com/pmodels/yaksa/commit/ef90cf608bd3f9c412e3b804653e00ef4c9d29a5

Reply:
Oh, maybe that is already in `0.1x`?

Reply:
https://github.com/pmodels/yaksa/pull/169 -- Add `nogpu` info hint -- is also worth to mention.

Reply:
No, I see it in the shortlog. Added a line for it.

Reply:
> #169 -- Add `nogpu` info hint -- is also worth to mention.

[we discussed offline] This patch was actually included in the 0.1x branch used for MPICH 3.4.3 which is why I originally left it out, but I do think it is worth a mention on second thought. Will add.